### PR TITLE
Api request

### PR DESCRIPTION
**Pull request by jrsewell400**
# Configure Styling Component

Please include a summary of the change:
* Adds functionality that allows FE to make a call using the '/results' route, and will return a JSON response with all necessary information minus savings information. A future iteration will include savings information.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Note any questions or struggles as well.
* Phase 2, need to refactor to only return 'outputs' portion of API response and will need to add additional API call for savings information.

## Type of change
Please uncheck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Add Testing
- [ ] Refactor

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: test/test_app.py [hunk 1]
```diff
@@ -1,4 +1,88 @@
...
+    assert response_body["outputs"]["solrad_monthly"][2] == 5.921163558959961
+    assert response_body["outputs"]["ac_annual"] == 6605.365234375
+    assert response_body["outputs"]["ac_monthly"][0] == 479.64813232421875
+    assert response_body["outputs"]["ac_monthly"][1] == 493.1632385253906
+    assert response_body["outputs"]["ac_monthly"][2] == 599.2940673828125
+
+    # pretty_print_request(response.request)
+    # pretty_print_response(response)
+
+
```
Comment (nickstaylor) [inline test/test_app.py:66]: Looks like all of the input parameters are accounted for in this test. Is there a test for the sad path if the parameters don't fit the requirements?  Nicely done!

File: app.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        self.system_capacity = system_capacity
+        self.azimuth = azimuth
+        self.tilt = tilt
+        self.array_type = array_type
+        self.module_type = module_type
+        self.losses = losses
+
+    def get(self):
+        r = requests.get(f'https://developer.nrel.gov/api/pvwatts/v6.json?api_key={os.getenv("NREL_API_KEY")}&address={self.address}&system_capacity={self.system_capacity}&azimuth={self.azimuth}&tilt={self.tilt}&array_type={self.array_type}&module_type={self.module_type}&losses={self.losses}')
+        # r = requests.get(f'https://developer.nrel.gov/api/pvwatts/v6.json?api_key={os.getenv("NREL_API_KEY")}&address=322+N+7th+ave+beech+grove+in&system_capacity=4&azimuth=180&tilt=40&array_type=1&module_type=1&losses=10')
```
Comment (PaulDebevec) [inline app.py]: We're going to want to remove this commented code here pre-merge

File: app.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        print(r.json())
+
+nrel = NREL("322+N+7th+ave+beech+grove+in", 4, 180, 40, 1, 1, 10)
```
Comment (PaulDebevec) [inline app.py]: I think we can also clean up 22 & 23 for production.

File: config.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+class Config(object):
+    DEBUG = False
+    CSRF_ENABLED = False
+    # SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/tasks'
```
Comment (PaulDebevec) [inline config.py:4]: I look forward to getting the DB in here to play around with this.

File: app.py [hunk 5]
```diff
@@ -1,37 +1,42 @@
-from flask import Flask, request, jsonify
+# from flask_sqlalchemy   import SQLAlchemy
+# from flask_marshmallow import Marshmallow
+# import code
+# code.interact(local=dict(globals(), **locals()))
```
Comment (PaulDebevec) [inline app.py:4]: We might consider pulling these comments here out into a google doc

File: app.py [hunk 6]
```diff
@@ -1,37 +1,42 @@
...
-# Get NREL solar system data
-@app.route('/results/<fe_json>', methods=['GET'])
-def get_solar_system(fe_json):
-    results = NREL(fe_json)
-    return results.get()
 
+# Get solar system data from NREL API
+@app.route('/results', methods=['POST'])
+def get_solar_system():
+    results = NREL(request.json)
```
Comment (PaulDebevec) [inline app.py:20]: Awesome job getting this to work!

File: test/test_app.py (hunk 1 above)
Comment (PaulDebevec) [reply test/test_app.py:66]: I don't think we have sad path integrated just yet. It is however on the project board to add that testing.

Comment (jrsewell400) [reply test/test_app.py:66]: What Paul said!

File: app.py (hunk 2 above)
Comment (jrsewell400) [reply app.py]: I'm seeing an "outdated" tag next to app.py (not sure what that means exactly), and I don't see this syntax in the committed file.  I think we should be good to go here.

File: app.py (hunk 3 above)
Comment (jrsewell400) [reply app.py]: Same as the comment above, seeing outdated and not seeing it in the committed code.  Think we should be okay.

File: app.py (hunk 5 above)
Comment (jrsewell400) [reply app.py]: Done!

File: app.py (hunk 6 above)
Comment (jrsewell400) [reply app.py:16]: Thanks man!  I appreciate the help in getting to this point!

File: app.py (hunk 2 above)
Comment (PaulDebevec) [reply app.py]: I think you're right there my friend.

File: app.py (hunk 3 above)
Comment (PaulDebevec) [reply app.py]: Right on - I'll keep an eye out for this in the future! I don't recall seeing this tag before.

**Pull request merged**